docs/kqueen.cli_azure_examples.rst

```diff
@@ -0,0 +1,116 @@
+Azure CLI operations with KQueen
+---------------------------------------
+
+In KQueen, you can perform a number of Azure operations using the command-line interface (CLI).
+For example, you can create a Azure account, create a new Kubernetes cluster (AKS), download the kubeconfig and use it to push an application into Kubernetes.
+
+
+To create / configure the Azure to work with KQueen:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Login to your Azure portal:
+
+.. code-block:: bash
```

Comment:
Please indent all code blocks to be directly under the text in the numbered list. For example:
#. Log in to your Azure portal:

    .. code-block::

       ...

Reply:
I'm not getting it - can you give me some more examples please?

Reply:
Sure, see line 186: https://github.com/Mirantis/kqueen/pull/299/files/d2e97c6932f7341f62001b8680717da20228b704#diff-b7d1f99034378966a34db3278f4a3cf9
the code block is right under the text. So you need to add more spaces for the code block and its text.

Otherwise, the numbered list is broken: https://github.com/ruzickap/kqueen/blob/8476e2481d294632e9e705f9adf9d8c0f4030a1e/docs/kqueen.cli_azure_examples.rst (we have step 1 everywhere).